baseplate/metrics.py

```diff
@@ -261,6 +272,24 @@ def decrement(self, delta=1, sample_rate=1.0):
         """
         self.increment(delta=-delta, sample_rate=sample_rate)
 
+class Histogram(object):
+    """Record of arbitrary values for distribution over time.
```

Comment:
record of?

Reply:
Yeah, I'm kinda stuck on this wording. i wanted it to be a noun as this is a class, but this does read clunkily. Any ideas?

Reply:
I was thinking just "Distribution of arbitrary values over time" maybe? In my mind this was technically a record of said distribution.

Reply:
I'll change to `A record`

Reply:
I'm wondering if the naming involved can take some hints from other histogram projects, e.g. https://github.com/HdrHistogram/HdrHistogram/blob/master/src/main/java/org/HdrHistogram/AbstractHistogram.java

Reply:
(I totally misread this originally as the verb "record" so "record of" just sounded really off)

Reply:
This interface is sorta RPCey so maybe we just treat this histogram object as the histogram as exists in statsite. Then the description is along the lines of just describing what a histogram in general is.

Reply:
I like that.